docs/FAQ.md

```diff
@@ -3,49 +3,102 @@ title: eScriptorium Tutorial - FAQ
 summary: Frequently asked questions about eScriptorium.
 authors:
     - Alix Chagué
-date: 2023-01-31
+date: 2023-02-22
 ---
 
 # Frequently asked questions about eScriptorium
-!!! warning
-    The section needs to be completed.
 
+???+ Tip "Missing answers!"
+    Feeling like we missed an important question? See how to [contribute](contribute.md) to this tutorial and submit your idea!
+
+[TOC]
+
+---
 ## How can I have an eScriptorium account?
 
-## Who is responsible for eScriptorium?
+!!! Note
+    The text in this section cites eScriptorium's FAQ: 
+    
+    ➡️ [https://gitlab.com/scripta/escriptorium/-/wikis/How-can-I-get-an-account-on-eScriptorium%3F](https://gitlab.com/scripta/escriptorium/-/wikis/How-can-I-get-an-account-on-eScriptorium%3F)
 
-See: https://gitlab.com/scripta/escriptorium/-/wikis/Who-is-responsible-for-eScriptorium%3F
+> eScriptorium is provided as Free and Open Source Software, not as a service. This means that there is no one eScriptorium site, but instead there are many different servers running the software which have been set up by different people in different institutions. To use eScriptorium, then, you can either set up an installation yourself (for which see Deploying eScriptorium) or find an existing one and ask the owners for access. Unfortunately we don’t know all the different installations because, again, eScriptorium is entirely free and open source, so people can and do install it without any connection to us and often without even telling us that they are using it.
```

Comment:
For "Deploying eScriptorium" --> could you put a direct link to this specific part in the Gitlab to help the user ?!